.github/workflows/validate_kong_image.yaml

```
@@ -0,0 +1,68 @@
+name: "validate a Kong version (targeted)"
```

```suggestion
name: "validate Kong image (targeted)"
```
Just to follow the file name.